src/matter/cluster/WaterContentMeasurementCluster.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {Attribute, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse} from "./Cluster";
+import {BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8} from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for WaterContent Measurement.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.6
+ */
+/** Percentage of water in the air */
+export const RelativeHumidityCluster = Cluster({
+    id: 0x0405,
+    name: "RelativeHumidityMeasurement",
+    revision: 3,
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.6.3 */
+    attributes: {
```

Comment:
Because of the fact that the attributes are 100% identical for all 3 clusters (right?) You can move them in an own "const attributes = { ... }" and simply use this then in the definition

Reply:
Thanks - I knew there must be a way to do that. I'll eventually get the hang of Typescript.